src/Core/Houston.vala

```diff
@@ -17,7 +17,11 @@
 */
 
 public class AppCenterCore.Houston : Object {
+#if POP_OS
+    private const string HOUSTON_API_URL = "https://api.pop-os.org/shop/v1";
```

Comment:
There are a lot of instances of changing constants in files throughout this diff. I'd be tempted to suggest making a separate `Constants.vala` file here in upstream. I think we'd be open to that with the justification of it making it much easier for downstreams to pull in changes. As that file would be unlikely to change, you'd be a lot less likely to have merge conflicts when pulling in upstream changes.